account_customer_wallet/models/product_template.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (C) 2022-Today: GRAP (http://www.grap.coop)
+# @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+
+    is_customer_wallet_product = fields.Boolean(
+        string="Wallet Product",
+        help="Check this box if this product is used to credit"
+        " customer wallets. Important note : you should set the"
+        " the same income and expense account as the journal wallet.",
```

Comment:
Same comment as above: can we do this automagically?

Reply:
same remark as here : https://github.com/coopiteasy/addons/pull/257#discussion_r963002859

Reply:
Thinking again on that topic, I think that try to automagically configure product / journal will generate a lot of code. Not sure it's a good tradeoff.
(We should overload the create, the write, add a constraint)